app/src/main/java/app/grapheneos/camera/capturer/ImageCapturer.kt

```diff
@@ -25,11 +25,14 @@ import app.grapheneos.camera.ui.activities.SecureMainActivity
 import app.grapheneos.camera.util.printStackTraceToString
 
 private const val imageFileFormat = ".jpg"
-var isTakingPicture: Boolean = false
 
 class ImageCapturer(val mActivity: MainActivity) {
     val camConfig = mActivity.camConfig
 
+    var isTakingPicture: Boolean = false
```

Comment:
isTakingPicture variable is redundant, it's always false when currentImageSaver is null and always true when currentImageSaver is non-null.

Reply:
Yes, we could set currentImageSaver to null instead of setting a separate variable. I'll push changes for the same soon